docs/index.rst

```diff
@@ -0,0 +1,24 @@
+.. bioconda-recipes documentation master file, created by
```

Comment:
Yeah, but I'll leave this up to you? You have a better idea of what should be on the front page.